packages/editor-ui/src/mixins/workflowRun.ts

```diff
@@ -175,43 +175,13 @@ export const workflowRun = defineComponent({
 
 				const runData = this.workflowsStore.getWorkflowRunData;
 
-				let newRunData: IRunData | undefined;
+				const newRunData = Object.entries(runData || {}).reduce((acc, [key, value]) => {
```

Comment:
can you explain how this fixes the issue? maybe add a comment?

Reply:
The frontend wasn't sending proper `runData` when executing nodes individually